specklepy/objects/base.py

```diff
@@ -403,3 +410,45 @@ class DataChunk(Base, speckle_type="Speckle.Core.Models.DataChunk"):
 
     def __init__(self) -> None:
         self.data = []
+
+    @classmethod
```

Comment:
I don't think the `DataChunk` should contain these methods. the `DataChunk` is just a dumb class whose purpose is to hold data. the serialisation / deserialisation is what deals with chunking and dechunking of lists without any relation to the data models. the `DataChunk` shouldn't deal with creating / deserialising objects itself.

Reply:
Resolved in 0270219